infrastructure/azure-pipelines-dev.yml

```diff
@@ -8,38 +8,80 @@ resources:
       ref: refs/heads/main
 variables:
   phpVersion: "8.2"
-
+  serviceConnectionName: "Talent Cloud Service Connection"
+  npm_config_user_config: "$(System.DefaultWorkingDirectory)/.npmrc"
+  dirResults: "$(System.DefaultWorkingDirectory)/apps/playwright/test-results"
+  resourceGroupName: "rg_tbs_talent_2_PaaS"
+  vertical: "dev"
+  storageAccountName: "sttalclddevcacstdl"
+  storageContainer: "playwright-results"
```

Comment:
Does this container automatically clean up old results?

Reply:
we have set a lifecycle rule on the container level to delete older results. Currently it is more than 30 days old.